test_elasticsearch_async/test_connection.py

```diff
@@ -58,7 +58,7 @@ def test_error_is_properly_logged(connection, caplog, port, server):
 def test_timeout_is_properly_raised(connection, server):
     @asyncio.coroutine
     def slow_request():
-        yield from asyncio.sleep(0.01)
+        yield from asyncio.sleep(1)
```

Comment:
It would be ideal to only have this PR touch the parts needed to upgrade to aiohttp >= 2.x and not touch other parts of the code or constructs.

We can then have separate PRs for those if those are not ideal, but one step at a time please. This is changing too many things at once.